src/utils/wallet.js

```diff
@@ -87,16 +108,91 @@ class Wallet {
         this.accountId = localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID) || ''
     }
 
+    /********************************
```

Comment:
@mattlockyer why not move these into separate module for 2FA?

Reply:
I will work on this, but each case is unique and references the wallet instance. Each component / action that imports will have to also get the instance of the wallet and then call the respective function.

I know this is a bit messy, perhaps you can suggest a better approach for decoupling?